data_store/rentals_data.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module PersistRental
+  def store_rentals(rentals)
+    rental_array = []
+    file = './json_files/rentals.json'
+    return unless File.exist?(file)
+
+    rentals.each do |rental|
+      rental_array << { date: rental.date, person: rental.person, book: rental.book }
+    end
+    File.write(file, JSON.generate(rental_array))
+  end
+
+  def fetch_rentals
+    rental_array = []
+    file = './json_files/rentals.json'
+    return rental_array unless File.exist?(file) && File.read(file) != ''
+
+    JSON.parse(File.read(file)).each do |rental|
+      rental_array << Rental.new(rental['date'], rental[person], rental[book])
+    end
+
+    rental_array
+  end
```

Comment:
- [ ] kindly fix the issue when there is some data in rentals.json, the following error raise 
>![image](https://user-images.githubusercontent.com/53402600/188892050-c437229c-2efd-4379-9b1b-cc0661d736a1.png)